### PR TITLE
Image Transition selector

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Add a selector for picking Image component transition type.
Keep selector hidden while feature is not implemented in the component.

## Motivation and Context
UX Improvements epic.

## How Has This Been Tested?
Tested on stage-19 with visible and stage-1 with hidden selector.

Visible: http://apps-stage-19.risevision.com/templates/edit/9e929d12-536a-4492-a095-2448b3e8eeee/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443
Hidden: http://apps-stage-1.risevision.com/templates/edit/9e929d12-536a-4492-a095-2448b3e8eeee/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu @stulees The last commit is what needs to be reverted once the feature is implemented. Apart from making the element visible, there is a styling change due to the different height.
Please review. Thanks!

File: web/scss/sections/components/image.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
 @media (min-width: $screen-sm) {
   .image-component-container {
     --image-component-list-button-bar-height: 115px;
-    --image-component-list-duration-height: 155px;
+    --image-component-list-duration-height: 80px;
```
Comment (alex-deaconu) [inline web/scss/sections/components/image.scss]: @ezequielc wondering if you should make a second variable for this (`image-component-list-transition-height`), or re-name this variable to make it reflect the fact that the height includes both duration and transition. Thanks.

Comment (ezequielc) [reply web/scss/sections/components/image.scss]: @alex-deaconu Good suggestion. I will rename it to `image-component-list-attributes-height`. Thanks

**Comment by ezequielc** [comment]
@stulees Yeah, I don't know the answer. I got the transitions from the [JTBD](https://docs.google.com/document/d/11kwU1GV2VnIlz2Y8bKmwqTrLDASkUJV_bgKiA9DovNk/edit#bookmark=id.vctgqu78arw). I assume you can revise them with Shea as part of the implementation epic and adjust here accordingly. Thanks!

**Pull request merged**